tests/test_serialization.py

```diff
@@ -105,3 +105,26 @@ def test_pickle_list() -> None:
         fp.seek(0)
         c1 = pickle.load(fp)
         assert c == c1
+
+
+def test_load_duplicate_keys_top() -> None:
+    from yaml.constructor import ConstructorError
+
+    try:
+        with tempfile.NamedTemporaryFile(delete=False) as fp:
+            fp.write("a:\n  b: 1\na:\n  b: 2\n".encode("utf-8"))
```

Comment:
Switch tests to use """ strings to to make them easier to understand.
"""
```suggestion
            content = """
a:
  b: 1
a:
  b: 2
"""
            fp.write(content.encode("utf-8"))
```